src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildWrapper.java

```diff
@@ -792,12 +803,12 @@ public SauceConnectCloser call() throws AbstractSauceTunnelManager.SauceConnectE
          * @param workingDirectory
          * @param resolvedOptions
          */
-        public SauceConnectHandler(SauceOnDemandBuildWrapper sauceOnDemandBuildWrapper, BuildListener listener, String workingDirectory, String resolvedOptions) {
+        public SauceConnectHandler(SauceOnDemandBuildWrapper sauceOnDemandBuildWrapper, EnvVars env, BuildListener listener, String workingDirectory, String resolvedOptions) {
```

Comment:
@oleg-nenashev I've searched and do not see any uses of this constructor outside of SauceOnDemandBuildWrapper (both within the plugin and all of GitHub: https://github.com/search?q=SauceConnectHandler&type=Code)

Is there an acceptable scenario to skip the extra constructor? I'd rather note further pollute the codebase and leave a never-to-be-removed @deprecated constructor I see often. Thoughts?